types/error.example.json

```diff
@@ -1,5 +1,14 @@
 {
   "@type": "Error",
-  "identifier": "1",
-  "message": "Conflicting arrival date"
+  "airline": "Spirit Airline",
+  "errorLevel": "error",
```

Comment:
errorLevels should be capitalized, e.g. ERROR, WARNING because they are standard in the industry